app/controllers/tag_controller.rb

```diff
@@ -8,6 +8,7 @@ def index
     @title = I18n.t('tag_controller.tags')
     @paginated = true
     @order_type = params[:order] == "desc" ? "asc" : "desc"
+    powertag_clause = current_user&.admin? ? '' : ['name NOT LIKE ?', '%:%']
```

Comment:
If you want to take one more step, we could make this be dependent on a parameter instead, like `params[:powertags]` -- that way, anyone could use it, but only by using the URL `/tags?powertags=true` -- what do you think?

Reply:
No problem, switched from a user role check to a param check. Let me know if I should have kept the admin one as well.
As expected, pagination is ok but `powertags=true` is lost if tags sort criteria changes and has to be put into the URL again. Should I fix it in [tag view](https://github.com/publiclab/plots2/blob/8ca4b786fdc895e6c2c3e06529a394c38b959a8d/app/views/tag/index.html.erb#L23-L25) or is a UI powertags button planned/preferred? 
Thanks!

Reply:
Hi @simonastefan, I think it would be good to fix in the Tag view only as we don't want to have separate buttons for power tags. @jywarren what do you think? Thanks!

Reply:
Updated with view fix